### PR TITLE
Update README.md, PR related to this issues #4250

### PR DESCRIPTION
**Pull request by leonyangela**
## What does this PR do?
Added a tutorial on how to create a demo app with appwrite + jquery, I add a dev.to post link and include my github repo on the post.

## Test Plan

To run the app install the app by using `npm install`, then `npm run dev`.

## Related PRs and Issues

this PR is related to this issues [#4250](https://github.com/appwrite/appwrite/issues/4250).

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)? 
Yes, I've read the Contributing guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +162,7 @@ The Almost Netflix series is a tutorial for building a Netflix clone with Appwri
 * [How To Generate and Store Map Previews in Ruby with Appwrite](https://dev.to/oliverpham/how-to-generate-and-store-map-previews-in-ruby-with-appwrite-43ha)
 * [What is Appwrite and who i setup in my project?](https://dev.to/jordi5107/what-is-appwrite-and-who-i-setup-in-my-project-3af)
 * [How Appwrite real-time can make your CMS changes instant](https://sakshiuppoor.hashnode.dev/how-appwrite-real-time-can-make-your-cms-changes-instant)
+* [Appwrite + JQuery](https://dev.to/leonyangela/build-your-app-with-appwrite-jquery-55ij)
```
Comment (stnguyen90) [inline README.md:165]: Great PR! 🤯 A few requests:

1. Would you please update the README in your repo to include instructions about the collection attribute needed?
2. Would you please also update the README to include some screenshots?

Comment (leonyangela) [reply README.md:165]: Hi Steven, sure. I just added the instructions and the screenshots. Thank you :).

**Comment by leonyangela** [comment]
@stnguyen90 great to hear that, thanks steven!

**Comment by stnguyen90** [comment]
@leonyangela, we'll need the PR open to be able to merge it 😉

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**